index.html

```diff
@@ -13,31 +13,32 @@
       <h1 class="header" id="#top">
         <span>Get to know...</span> <span>Julia Schneider</span>
       </h1>
-      <nav class="box-styling list navigation">
-        <a class="button" href="#interests">Interests</a>
-        <a class="button" href="#skills">Skills</a>
-        <a class="button" href="#experience">Experience</a>
-        <a class="button" href="#contact">Contact</a>
-        <a class="button" href="#biography">Biography</a>
-        <a class="button" href="#follow-me-on">Follow me on</a>
-        <a class="button" href="#cv">CV</a>
+      <nav class="section__list navigation">
+        <a class="navigation__link--font" href="#interests">Interests</a>
+        <a class="navigation__link--font" href="#skills">Skills</a>
+        <a class="navigation__link--font" href="#experience">Experience</a>
+        <a class="navigation__link--font" href="#contact">Contact</a>
+        <a class="navigation__link--font" href="#biography">Biography</a>
+        <a class="navigation__link--font" href="#follow-me-on">Follow me on</a>
+        <a class="navigation__link--font" href="#cv">CV</a>
       </nav>
```

Comment:
I would recommend using the same styles twice and stick to the naming convention with BEM